



>>> quinta-feira, 12 de novembro de 2020 às 13:35 <<<

> <:emojicoffeeinfo:766020288885293097> Aviso!
> Alguns commandos foram atualizados no banco de dados!
> Request: {
    "aliases": [
        {
            "id": 53,
            "value": [
                "reportar"
            ]
        }
    ],
    "releases_notes": [
        {
            "id": 53,
            "value": {
                "1.0": null
            }
        }
    ]
}
> Resposta: {
    "result": {
        "fieldCount": 0,
        "affectedRows": 1,
        "insertId": 0,
        "serverStatus": 2,
        "warningCount": 0,
        "message": "(Rows matched: 1  Changed: 0  Warnings: 0",
        "protocol41": true,
        "changedRows": 0
    },
    "sql": "UPDATE commands SET aliases = CASE id WHEN 53 THEN '[\"reportar\"]' ELSE aliases END, releases_notes = CASE id WHEN 53 THEN '{\"1.0\":null}' ELSE releases_notes END WHERE id IN (53)",
    "commands": [
        {
            "id": 53,
            "name": "report",
            "aliases": [
                "reportar"
            ],
            "type": "Contato",
            "description": "Se você encontrou algum bug ou coisa que você acredita que não esteja funcionando como deveria, basta usar o comando para que os devenvolvedores fiquem a par do problema e o resolva.",
            "how_to_use": "Digite _report no chat e passe como parâmetro o seu report, em seguida envie a mensagem.",
            "cooldown": 10000,
            "example": "_report Encontrei um bug ao executar o comando ajuda, ao usar o comando o bot não responde!",
            "example_url": null,
            "created_timestamp": 1598646020000,
            "updated_timestamp": 1603328174440,
            "version": "1.0",
            "releases_notes": {
                "1.0": null
            },
            "times_limit": 2,
            "active": 1,
            "reason_inactivity": "EM FASE DE TESTES"
        }
    ]
}



>>> quinta-feira, 12 de novembro de 2020 às 13:35 <<<

> <:emojicoffeeerro:766020289074167828> Erro
> Aconteceu um problema ao fazer a requisição de deletar itens do inventorio que estava vencidos para a api.
> Os IDs das items: [
    32
]
> O erro: "{
    "message": "Request failed with status code 400",
    "name": "Error",
    "stack": "Error: Request failed with status code 400\n    at createError (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\createError.js:16:15)\n    at settle (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\core\\settle.js:17:12)\n    at IncomingMessage.handleStreamEnd (C:\\Users\\Pichau\\Documents\\Bots\\Coffe\\node_modules\\axios\\lib\\adapters\\http.js:244:11)\n    at IncomingMessage.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)",
    "config": {
        "url": "/inventory/delete",
        "method": "delete",
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjczOTMwOTMwNTU4ODc0NDI2MyIsImlhdCI6MTYwNTE5ODkzNH0.0M5Oxt3p8CNTtncJbjSDeTgyjbo9w9gGp2ruMhDkRtk",
            "User-Agent": "axios/0.20.0"
        },
        "baseURL": "http://localhost:3001/api",
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1,
        "body": {
            "properties": {
                "id": [
                    32
                ]
            }
        }
    }
}
{
    "error": "Missing properties"
}"